src/plugins/bsodmon/bsodmon.cpp

```diff
@@ -205,3 +206,17 @@ bsodmon::bsodmon(drakvuf_t drakvuf, bool _abort_on_bsod, output_format_t output)
     if ( !drakvuf_get_kernel_symbol_rva( drakvuf, "KeBugCheck2", &trap.breakpoint.rva) ) throw -1;
     if ( ! drakvuf_add_trap( drakvuf, &trap ) ) throw -1;
 }
+
+bsodmon::~bsodmon()
+{
+}
+
+void bsodmon::stop()
+{
+    drakvuf_remove_trap(drakvuf, &trap, nullptr);
+}
+
+bool bsodmon::is_stopped()
+{
+    return true;
```

Comment:
This doesn't make much sense. Shouldn't this return true if it is actually stopped, ie. `stop()` was called on it already?

Reply:
This make sense. Thank You!